index.js

```diff
@@ -1,39 +1,267 @@
-const readline = require('readline');
-const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+/*
+BOILERPLATE STUFF DO NOT TOUCH ++++++++++++++++++++++++++++++++++++++++++++++++++
+*/
+const readline = require("readline");
+const readlineInterface = readline.createInterface(
+  process.stdin,
+  process.stdout
+);
 
 function ask(questionText) {
   return new Promise((resolve, reject) => {
     readlineInterface.question(questionText, resolve);
   });
 }
+/*
+END BOILERPLATE STUFF +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+*/
 
-// remember the StateMachine lecture
-// https://bootcamp.burlingtoncodeacademy.com/lessons/cs/state-machines
-let states = {
-  'roomOne': { canChangeTo: [ 'roomTwo' ] },
-  'roomTwo': { canChangeTo: [ 'roomThree' ] },
-  'roomThree': { canChangeTo: [ 'roomOne' ] }
-};
-
-let currentState = "green";
-
-function enterState(newState) {
-  let validTransitions = states[currentState].canChangeTo;
-  if (validTransitions.includes(newState)) {
-    currentState = newState;
-  } else {
-    throw 'Invalid state transition attempted - from ' + currentState + ' to ' + newState;
+//DO NOT CHANGE MYFUNK
+myFunk();
+async function myFunk() {
+  //OBJECTS GO HERE!!!!!!!!!!!!!
+  let inv = {
+    //IGNORE FOR NOW
+  };
+  let userStatus = {
+    //IGNORE FOR NOW
+  };
+  function showInv() {
+    for (const key in inv) {
+      if (inv.hasOwnProperty(key)) {
+        const element = inv[key];
+        console.log(element);
+      } else {
+        console.log("you have nothing in your inventory!");
+      }
+    }
   }
-}
 
-start();
-
-async function start() {
-  const welcomeMessage = `182 Main St.
-You are standing on Main Street between Church and South Winooski.
-There is a door here. A keypad sits on the handle.
-On the door is a handwritten sign.`;
-  let answer = await ask(welcomeMessage);
-  console.log('Now write your code to make this work!');
-  process.exit();
+  /*
+FIRST ROOM ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+*/
+  begin();
+  //TO DO: THIS WILL BE OUR START STATE, WITH KEYPAD, SIGN, ETC.
+  async function begin() {
+    let firstLoop = true;
+    let answer = await ask(
+      "You are standing on Main Street between Church and South Winooski. There is a door here. A keypad sits on the handle. On the door is a handwritten sign. \n"
+    );
+    while (firstLoop === true) {
```

Comment:
You don't actually need to evaluate `firstLoop` here since it is a boolean, you could just say `while(firstLoop)` this holds true for all your loops